Source/WebKit/UIProcess/SystemPreviewController.h

```diff
@@ -70,10 +71,22 @@ class SystemPreviewController : public CanMakeWeakPtr<SystemPreviewController> {
     void setCompletionHandlerForLoadTesting(CompletionHandler<void(bool)>&&);
 
 private:
-
     void takeActivityToken();
     void releaseActivityTokenIfNecessary();
 
+    NSArray *localFileURLs() const;
+
+    enum class State : uint8_t {
```

Comment:
Do we anticipate uses for this State outside of the check for loading? If not we could also just make this a binary flag?

Reply:
I do expect we'll want the other states (but yes, I did briefly consider making just a bool).